### DISCUSSION TITLE
Outbound SMS

### DISCUSSION THREAD

**Discussion by stackboostio** [twilio-professional-services/flex-project-template#533] [General]
Something I'm really struggling with is outbound SMS.

I saw the repo for outbound-messaging, but it looks like it's in Node14 and no longer viable.  

Are there any thoughts on how to best approach this, or is it being considered as a plugin?  It seems under-discussed or maybe unimportant.  

Thanks :)

**Comment by dremin** [comment]
Hi @stackboostio!

Thanks for bringing it to our attention that the [plugin-outbound-cbm repository](https://github.com/twilio-professional-services/plugin-outbound-cbm) was out of date! I have updated it with the latest dependencies, so you should be able to successfully deploy that now.

**Comment by stackboostio** [comment]
Wow thanks for the quick reply - there was another fork that was already updated, so I tried that and successfully deployed it, but I think it conflicts with the dialpad on the "big" plugin.  I'll give yours a shot too
